website/docs/developer/deployment-guide/recovery-workflow.mdx

```diff
@@ -34,3 +34,251 @@ directly or by checking the status on the Cumulus Dashboard.
     zoomOutPic={useBaseUrl('img/zoom-out.svg')}
     resetPic={useBaseUrl('img/zoom-pan-reset.svg')}
 />
+
+See the recovery workflow [stepfunction modules](https://github.com/nasa/cumulus-orca/tree/master/modules/workflows/OrcaRecoveryWorkflow) 
+for additional details on recovering files belonging to a granule.
+
+## Recovery workflow input and output examples
```

Comment:
Talked this over with @rizbihassan, and thought we could just have this information in the operator doc and link to it here so we don't have duplicates. @bhazuka, your thoughts?

Reply:
I think that sounds like a good idea to me.

Reply:
will add a link to the other doc